controlnet3d.py

```diff
@@ -91,7 +80,7 @@ class ControlNet3DConditioningEmbedding(nn.Module):
     def __init__(
         self,
         conditioning_embedding_channels: int,
-        conditioning_channels: int = 3,
+        conditioning_channels: int = 2,
```

Comment:
Does this matter? I thought we end up passing in the right value anyway?

Reply:
No it doesn't matter, but it confused me for a moment, so I thought I might as well update it there as well. This still needs to be passed in correctly from the train script.